js/clinicalTimeline.js

```diff
@@ -261,6 +283,139 @@ window.clinicalTimeline = (function(){
   }
 
   /**
+   * Hanles the drawing and panning of the overviewAxis
+   */
+  function handleOverviewAxes() {
```

Comment:
Name of this function is plural (axes), whereas you prolly want the singular (axis)